test/fail_compilation/ice15002.d

```diff
@@ -1,8 +1,9 @@
 /*
 TEST_OUTPUT:
 ---
-fail_compilation/ice15002.d(10): Error: array index 5 is out of bounds `x[0 .. 3]`
-fail_compilation/ice15002.d(10): Error: array index 5 is out of bounds `x[0 .. 3]`
+fail_compilation/ice15002.d(10): Deprecation: mismatched array lengths. array initializer has 0 elements, but array length is 3
```

Comment:
In this case, just remove the initializer on line 10, it should not affect the test.